### PR TITLE
5398 - Remove Audioswitch

### PR DESCRIPTION
**Pull request by jagamypriera**
This PR updated Twilio to newest version (`2.1.0`), removed AudioSwitch

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -175,6 +182,9 @@ declare module "react-native-twilio-video-webrtc" {
       videoBitrate?: number;
     };
     enableRemoteAudio?: boolean;
+    encodingParameters?: {
+      enableH264Codec?: boolean;
+    };
```
Comment (jamesogrant) [inline index.d.ts]: It looks like we're defining `encodingParameters` twice now?

**Review by jamesogrant — CHANGES_REQUESTED** [review]
LGTM except a duplicate definition of `encodingParameters` that needs removing.

File: index.d.ts (hunk 1 above)
Comment (jagamypriera) [reply index.d.ts]: ah right 🤦 
removing it.

**Review by jamesogrant — APPROVED** [review]
LGTM!

**Pull request merged**